installer/provision_el.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Install EPEL
+  package:
+    name: epel-release
+    state: installed
+
+- name: Install dependencies
+  package:
+    name:
+      - docker 
+      - gettext 
+      - python-pip 
+      - gcc-c++ 
+      - bzip2 
+    state: installed
+
+- name: Start and enable Docker
+  service:
+    name: docker
+    state: started
+    enabled: true
+
+- name: Upgrade pip
+  pip:
+    name: pip
+    extra_args: --upgrade
+
+- name: Install docker-py
+  pip:
+    name: docker-py
+
+- name: Get NodeJS repo
+  shell: "curl --silent --location https://rpm.nodesource.com/setup_6.x | sudo bash -"
```

Comment:
I'm nervous about implicitly piping some random url to sudo for everyone that runs this on their system. Perhaps take a look at @geerlingguy's role here? https://github.com/geerlingguy/ansible-role-nodejs

I'm open to other approaches too, the main thing I'd like to avoid is piping curl to bash.

Reply:
Right. That's why I mentioned above that I had planned on changing it. I just have to make sure I can get the right repos set up. I just wanted to get the PR in to make sure people actually wanted this before I do a ton of work to set everything up.